package.json

```diff
@@ -42,5 +41,8 @@
     "stylus": "^0.54.5",
     "tape": "^4.9.1",
     "tape-run": "^4.0.0"
+  },
+  "dependencies": {
+    "chart.js": "^2.9.2"
```

Comment:
This asshole brings in `moment` :( 

Maybe `TonicChart` should export some kind of function.

```js
module.exports = (Tonic, Chart) => {
  TonicChart.setChart(Chart)
  Tonic.add(TonicChart)
}
```

This way tonic components is zero dependency and the way you hook up charts requires a little bit more finesse.

Reply:
ah, yeah I agree. this is a huge dep.